snow/networking/sender/sender_test.go

```diff
@@ -1308,3 +1318,14 @@ func TestSender_Single_Request(t *testing.T) {
 		})
 	}
 }
+
+func createSubscriberAndChangeNotifier() (*block.ChangeNotifier, common.Subscription) {
+	var cn block.ChangeNotifier
+
+	subscription := func(ctx context.Context) (common.Message, error) {
+		<-ctx.Done()
+		return common.Message(0), ctx.Err()
+	}
+
+	return &cn, subscription
+}
```

Comment:
Same comment as in the router. Perhaps the noop subscription could be some test helper somewhere, but tbh it's so simple that I'm not concerned about the duplication.